functions.sh

```diff
@@ -240,8 +270,8 @@ efivar_write() {
 }
 
 efivar_read() {
-	efivar_setup "$1"
-	cat "$var" | tail -c +5
+	efivar_setup "${1:-}"
+	tail -c +5 "$var"
```

Comment:
I think this only *looked* like a "useless use of `cat`" -- the EFI var files are not normal and `tail` tries to `seek()` if its input is not `stdin`, which doesn't always work on `/sys` "files".

Reply:
Fixed.